src/main/java/bits/bits/team/command/CommandBeam.java

```diff
@@ -102,4 +106,26 @@ else if (argument.equalsIgnoreCase("cancel")) {
 
     return true;
   }
+
+  @Override
+  public List<String> onTabComplete(CommandSender commandSender, Command command, String alias, String[] args) {
+    if (!(commandSender instanceof Player)) return null;
+
+    List<String> result = new ArrayList<>();
+
+    if (args.length == 0) {
+      Bukkit.getOnlinePlayers().forEach(player -> result.add(player.getName()));
+    } else {
+      if (args[0].startsWith("a")) {
+        result.add("accept");
+      } else if (args[0].startsWith("c")) {
+        result.add("cancel");
+      }
+      Bukkit.getOnlinePlayers().stream()
```

Comment:
Always adds the matching players, instead of putting it in an else, which makes sure it still works for playernames starting with a 'c' or an 'a'.